querier.go

```diff
@@ -233,6 +240,7 @@ func (q *Querier) queryRange(ctx context.Context) {
 			Query: profileType,
 			Start: timestamppb.New(rangeStart),
 			End:   timestamppb.New(rangeEnd),
+			Step:  durationpb.New(time.Duration(tr.Nanoseconds() / numHorizontalPixelsOn8KDisplay)),
```

Comment:
The step parameter for QueryRange only depends on the start and end time in the query.
https://github.com/parca-dev/parca/blob/main/ui/packages/shared/utilities/src/time.ts#L129-L135

I think we need to adjust to use the `(end - start) / stepCount`. We can make our step count the `numHorizontalPixelsOn8KDisplay`.

Reply:
Thanks for the pointer! In this case `tr = end-start` so I think this does what's expected.